CHANGELOG.rst

```diff
@@ -5,6 +5,23 @@ Ansible Netcommon Collection Release Notes
 .. contents:: Topics
 
 
+v2.0.1
+======
+
+Minor Changes
```

Comment:
This has a minor change. Shouldn't the next version be 2.1.0?

Reply:
@NilashishC  so it doesn't look to be an impactful change here. let's wait for @Qalthos  inputs on this one.

Reply:
> This has a minor change. Shouldn't the next version be 2.1.0?

Going strictly by the labels, yes. I think that would be misleading though. The relicensing is an important change that should absolutely be called out and is in only the most tenuous sense a bugfix. But changing those licenses changed nothing about the functionality of the code and if anything only made it easier to use with the more permissive license.

In the end, I picked `Minor Changes` for this PR because changelogs are for humans, and this is a change, but not a particularly major one. We should definitely use the fragment sections to inform the expected next version, but in this case, I think we can bend it while humans are still picking the version numbers

Reply:
@Qalthos +1